Lanchonete/Lanchonete/ Features/Customization/Presentation/CustomizationViewPresenter.swift

```diff
@@ -7,22 +7,26 @@
 
 import Foundation
 
-public protocol CustomizationView: class {
+public protocol CustomizationView: AnyObject {
     func reloadData()
 }
+
+public protocol CustomizationViewDelegate: AnyObject {
+    func customizationViewDidEnd(with ingredients: [Ingredient])
+}
+
 public class CustomizationViewPresenter {
```

Comment:
Class does not have tests.

Reply:
Just like my previous answer, I won't be adding test for this presenter since I have assured that the presenter is covered by unit tests and the presenter and controller are showing the expected behaviour.